style.css

```diff
@@ -308,7 +876,7 @@ body {
   .skillList {
     width: 88vw;
     padding: 0.01%;
-    margin: 0.1vw 0.1vw 45vw 4%;
+    margin: 0.5vw 0.1vw 45vw 4%;
     display: flex;
```

Comment:
- [ ] [OPTIONAL] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` and assigning it wherever possible as a class name in your `html` markup code.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```
- And kindly attend to any repetitions in your code 🙂.
- DRY is a requirement but since the previous reviewer didn't take cognizance of it, I've made it optional.